satpy/etc/enhancements/mwr.yaml

```diff
@@ -0,0 +1,43 @@
+enhancements:
+
+  mw183_humidity_surface:
+    standard_name: mw183_humidity_surface
+    operations:
+    - name: inverse
+      method: !!python/name:satpy.enhancements.invert
+      args:
+      - [true, true, true]
+    - name: stretch
+      method: !!python/name:satpy.enhancements.stretch
+      kwargs: {stretch: linear}
+    - name: gamma
+      method: !!python/name:satpy.enhancements.gamma
+      kwargs: {gamma: 1.2}
+
+  mw325_humidity:
+    standard_name: mw325_humidity
+    operations:
+    - name: inverse
+      method: !!python/name:satpy.enhancements.invert
+      args:
+      - [true, true, true]
+    - name: stretch
+      method: !!python/name:satpy.enhancements.stretch
+      kwargs: {stretch: linear}
+    - name: gamma
+      method: !!python/name:satpy.enhancements.gamma
+      kwargs: {gamma: 1.2}
+
+  mw325_humidity_surface:
+    standard_name: mw325_humidity_surface
+    operations:
+    - name: inverse
+      method: !!python/name:satpy.enhancements.invert
+      args:
+      - [true, true, true]
+    - name: stretch
+      method: !!python/name:satpy.enhancements.stretch
+      kwargs: {stretch: linear}
+    - name: gamma
+      method: !!python/name:satpy.enhancements.gamma
+      kwargs: {gamma: 1.2}
```

Comment:
All of these three enhancements are identical, are they all needed? The corresponding composites could have the same `standard_name` so only one enhancement would be enough, `mw_humidty_surface` for example.

Reply:
If the enhancements are going to diverge in the future, that would be the time to add different enhancaments.